WebARonARKit/WebARonARKit/ViewController.h

```diff
@@ -17,42 +17,47 @@
 #import <ARKit/ARKit.h>
 #import <Metal/Metal.h>
 #import <MetalKit/MetalKit.h>
+#import <QuartzCore/QuartzCore.h>
 #import <UIKit/UIKit.h>
 #import <WebKit/WebKit.h>
-#import <QuartzCore/QuartzCore.h>
-
-@interface ViewController : UIViewController <WKUIDelegate, WKNavigationDelegate, UITextFieldDelegate, ARSessionDelegate, ARSessionObserver, WKScriptMessageHandler> {
 
+@interface ViewController
+: UIViewController<WKUIDelegate, WKNavigationDelegate, UITextFieldDelegate,
+ARSessionDelegate, ARSessionObserver,
+WKScriptMessageHandler> {
+    MTKView *mtkView; 
     WKWebView *wkWebView;
-
+    
     UITextField *urlTextField;
+    bool urlTextFieldActive;
     UIButton *backButton;
     UIButton *refreshButton;
-
+    
     bool initialPageLoadedWhenTrackingBegins;
-
+    
     UIDeviceOrientation deviceOrientation;
     UIInterfaceOrientation interfaceOrientation;
     bool updateWindowSize;
-
+    
     float near;
     float far;
-
+    
     bool showingCameraFeed;
+    bool iPhoneXDevice;
     UIColor *wkWebViewOriginalBackgroundColor;
-  
-    NSMutableDictionary* jsAnchorIdsToObjCAnchorIds;
-    NSMutableDictionary* objCAnchorIdsToJSAnchorIds;
-    NSMutableDictionary* anchors;
-  
+    
+    NSMutableDictionary *jsAnchorIdsToObjCAnchorIds;
```

Comment:
nit: I prefer the * next to the type because pointer is part of the type, not the symbol. YMMV I suppose.